scripts/app.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
+var projects = [];
 
-function Project(){};
+function Project(name, link){
+  this.name = name,
+  this.ref = link;
+};
 
-var getfit = new Project();
+var getfit = new Project('Get Fit', 'https://aesthetiques.github.io/swole_devs/');
+var busMall = new Project('Bus Mall', 'https://scottbrenden.github.io/bus-mall/');
+var salmonCookies = new Project('Salmon Cookies', 'https://scottbrenden.github.io/Salmon-cookies/');
+
+var projectObjects = [getfit, busMall, salmonCookies];
```

Comment:
Lines 9-13 are a very smart setup for now. I do advise as we move further into building your portfolio to get this arranged as object literals in a separate file similar to how starter code is.

https://github.com/codefellows-seattle-301d18/03-jQuery-and-events/blob/master/starter-code/scripts/blogArticles.js

Your life will be a lot easier doing it that way, but good job on this variation!